src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/RuntimeEnvironment.cs

```diff
@@ -34,6 +34,6 @@ public static string GetRuntimeDirectory()
         [Obsolete(Obsoletions.RuntimeEnvironmentMessage, DiagnosticId = Obsoletions.RuntimeEnvironmentDiagId, UrlFormat = Obsoletions.SharedUrlFormat)]
         public static object GetRuntimeInterfaceAsObject(Guid clsid, Guid riid) => throw new PlatformNotSupportedException();
 
-        public static string GetSystemVersion() => typeof(object).Assembly.ImageRuntimeVersion;
+        public static string GetSystemVersion() => "v" + Environment.Version;
```

Comment:
```suggestion
        public static string GetSystemVersion() => $"v{Environment.Version};
```
That will allocate a single string rather than first ToString'ing the Version to concatenate it with the 'v'.

I assume this isn't perf-sensitive enough to cache the result into a field to avoid repeated string creations on each access?

Reply:
> I assume this isn't perf-sensitive enough to cache the result into a field to avoid repeated string creations on each access?

I don't thnk so. For what its worth, the current call was also allocating a new string each time. I'm not sure this is ever a perf critical path and creating a field is probably more cost than just reallocating a string.